src/hooks/useFrozenScroll.js

```diff
@@ -0,0 +1,10 @@
+import {useContext} from 'react';
+import {ReportActionListFrozenScrollContext} from "../pages/home/report/ReportActionListFrozenScrollContext";
+
+/**
+ * Hook for getting current state of scroll freeze and a function to set whether the scroll should be frozen
+ * @returns {Object}
+ */
+export default function useFrozenScroll() {
+    return useContext(ReportActionListFrozenScrollContext);
```

Comment:
We could improve error handling here.

```suggestion
    const context = useContext(ReportActionListFrozenScrollContext);
    if (context === undefined)
        throw new Error('useFrozenScroll must be within ReportActionListFrozenScrollContextProvider');
    }
    return context;
```